packages/unity-js-bridge/src/thirdweb-bridge.ts

```diff
@@ -182,8 +182,8 @@ class ThirdwebBridge implements TWBridge {
     const walletInstance = this.walletMap.get(wallet);
     if (walletInstance) {
       if (walletInstance.walletId === "deviceWallet" && password) {
-        const deviceWallet = walletInstance as DeviceBrowserWallet;
-        await deviceWallet.connect({ chainId, password });
+        const deviceWallet = walletInstance as DeviceWallet;
+        await deviceWallet.connect({ chainId });
```

Comment:
how should we save/load in this flow? you're not using the password i pass in anymore

Reply:
I just made this change just to fix the build error for now, it's temporary and requires change.

We don't have automatic save anymore like we used to earlier. `save`, `load` needs to be done explicitly and password only required when encrypting

Reply:
modified the temporary code to this: 

```ts
// device wallet needs to be generated or loaded before connecting
if (walletInstance.walletId === "deviceWallet") {
  const deviceWallet = walletInstance as DeviceWallet;

  // if password is provided, we can load and save
  if (password) {
    if (await deviceWallet.isSaved()) {
      await deviceWallet.loadFromStorage({
        strategy: "encryptedJson",
        password,
      });
    } else {
      await deviceWallet.generate();
      await deviceWallet.save({
        strategy: "encryptedJson",
        password,
      });
    }
  }

  // if no password is provided, we can only generate
  else {
    await deviceWallet.generate();
  }
}

await walletInstance.connect({ chainId });
```